.vib/minio/cypress/cypress/integration/minio_test_suite.js

```diff
@@ -0,0 +1,57 @@
+/// <reference types="cypress" />
+import {
+  random,
+} from './utils'
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.get('[data-testid="ErrorOutlineIcon"]').should('not.exist');
+  cy.get('#logout').should('exist').click();
```

Comment:
```suggestion
  cy.get('#logout').should('be.visible').click();
```

Reply:
The element actually wasn't visible, so I used `exists`. However, I used `scrollIntoView` and now it's improved.